docs/reference/commandline/secret_create.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "secret create"
+description: "The secret create command description and usage"
+keywords: ["secret, create"]
+---
+
+<!-- This file is maintained within the docker/docker Github
+     repository at https://github.com/docker/docker/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# secret create
+
+```Markdown
+Usage:  docker secret create [NAME]
+
+Create a secret using stdin as content
+```
+
+Creates a secret using standard input for the secret content. You must run this
+command on a manager node.
+
+## Examples
+
+### Create a secret
+
+```bash
+$ cat ssh-dev | docker secret create ssh-dev
```

Comment:
In reality, isn't using stdin really insecure? Do we want to have a more best-practices kind of example?